src/mappings.rs

```diff
@@ -4,12 +4,12 @@ use crate::types::{BrickDesc, BrickMapping};
 use lazy_static::lazy_static;
 use regex::{Captures, Regex};
 use std::collections::{HashMap, HashSet};
+use brs::Direction::*;
 
 type RegexHandler = Box<dyn Fn(Captures, &bl_save::Brick) -> Option<BrickMapping> + Sync>;
 
 lazy_static! {
-    static ref BLANK_PRINTS: HashSet<&'static str> = vec![
-        "Letters/-space",
```

Comment:
What are the implications of no longer including `"Letters/-space"` here?

Reply:
The main difference between the prints `2x2f/blank`, etc and all `Letters/` prints is that the Letters have no border texture on their print surface whereas the blank prints have a border similar to the border texture for tile bricks in Brickadia. Since most prints including Letters don't have this tile texture they are now mapped by default to the SmoothTile asset when applied to a plate high print brick. Often times people would use the space letter print to hide the fact that they were using a ton of plate bricks for their interior floors/ceilings.

Reply:
> What are the implications of no longer including `"Letters/-space"` here?

![image](https://user-images.githubusercontent.com/3935931/141025233-83f87639-d6ed-4286-a90b-e0cae0302219.png)